modules/backend/assets/js/october-min.js

```diff
@@ -1250,4 +1250,32 @@ if(!!window.MSInputMethodContext&&!!document.documentMode){$(window).on('resize'
 fixSidebar()})
 function fixMediaManager(){var $el=$('div[data-control="media-manager"] .control-scrollpad')
 $el.height($el.parent().height())}
-function fixSidebar(){$('#layout-sidenav').height(Math.max($('#layout-body').innerHeight(),$(window).height()-$('#layout-mainmenu').height()))}}
\ No newline at end of file
+function fixSidebar(){$('#layout-sidenav').height(Math.max($('#layout-body').innerHeight(),$(window).height()-$('#layout-mainmenu').height()))}}
+function applyFocusVisiblePolyfill(scope){var hadKeyboardEvent=!0;var hadFocusVisibleRecently=!1;var hadFocusVisibleRecentlyTimeout=null;var inputTypesWhitelist={text:!0,search:!0,url:!0,tel:!0,email:!0,password:!0,number:!0,date:!0,month:!0,week:!0,time:!0,datetime:!0,'datetime-local':!0};function isValidFocusTarget(el){if(el&&el!==document&&el.nodeName!=='HTML'&&el.nodeName!=='BODY'&&'classList' in el&&'contains' in el.classList){return!0}
```

Comment:
Revert this file please, it still has the focus visible polyfill which should only be in storm.js now.

Reply:
Fixed.